### PR TITLE
more docs

### PR DESCRIPTION
**Pull request by sarob**
- updated readme
- mermaid workflows
- CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+3. Write clear commit messages
+4. Update documentation as needed
+5. Submit a pull request with a clear description
+
+## Code Style Guidelines
+
+- Follow PEP 8 standards for Python code
+- Include docstrings for new functions and classes
+- Add comments for complex logic
+- Maintain consistent naming conventions
```
Comment (surapuramakhil) [inline CONTRIBUTING.md:39]: can you add this clean sheet guide

- Follow this clean code sheetsheet - https://www.planetgeek.ch/wp-content/uploads/2014/11/Clean-Code-V2.4.pdf

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+- **enhancement**: New feature requests
+- **good first issue**: Good for newcomers
+- **help wanted**: Extra attention needed
+- **documentation**: Documentation improvements
+
+## Testing
+
+Before submitting a PR:
+
+- Test your changes thoroughly
```
Comment (surapuramakhil) [inline CONTRIBUTING.md:61]: Can you kindly add testing instructions.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Verify functionality with different configurations
+
+## Communication
+
+- Be respectful and constructive in discussions
+- Use clear and concise language
+- Reference relevant issues in commits and PRs
+- Ask for help when needed
+
+The project maintainers reserve the right to reject any contribution that doesn't meet these guidelines or align with the project's goals.
```
Comment (surapuramakhil) [inline CONTRIBUTING.md:73]: can you also add collaboration section -> Specify about Dev channel in Telegram


## 🚀 Join the AIHawk Community 🚀

Connect with like-minded individuals and get the most out of AIHawk.

💡 **Get support:** Ask questions, troubleshoot issues, and find solutions.

🗣️ **Share knowledge:** Share your experiences, tips, and best practices.

🤝 **Network:** Connect with other professionals and explore new opportunities.

🔔 **Stay updated:** Get the latest news and updates on AIHawk.


### Join Now 👇

[Telegram](https://t.me/AIhawkCommunity)

File: README.md [hunk 4]
```diff
@@ -670,37 +670,38 @@ For further assistance, please create an issue on the [GitHub repository](https:
...
+### For Users
 
-To install and configure **Ollama** and **Gemini**, please refer to the following documents:
+- Ollama & Gemini Setup
+  - To install and configure **Ollama** and **Gemini**, [Download Ollama and Gemini Setup Guide (PDF)](https://github.com/feder-cr/Auto_Jobs_Applier_AIHawk/blob/main/docs/guide_to_setup_ollama_and_gemini.pdf)
+  - Follow the instructions in these guides to ensure proper configuration of **AIHawk** with **Ollama** and **Gemini**.
+  - Written by Rushi, [Linkedin](https://www.linkedin.com/in/rushichaganti/), support him by following.
 
-- [Download Ollama and Gemini Setup Guide (PDF)](https://github.com/feder-cr/Auto_Jobs_Applier_AIHawk/blob/main/docs/guide_to_setup_ollama_and_gemini.pdf)
+- Editing YAML Files
```
Comment (surapuramakhil) [inline README.md:680]: kindly make this as title

File: README.md [hunk 5]
```diff
@@ -670,37 +670,38 @@ For further assistance, please create an issue on the [GitHub repository](https:
...
+  - For detailed instructions on editing YAML configuration sections for **AIHawk**, refer to this document:
+  - [Download YAML Editing Guide (PDF)](https://github.com/feder-cr/Auto_Jobs_Applier_AIHawk/blob/main/docs/guide_yaml_sections.pdf)
+  - Written by Rushi, [Linkedin](https://www.linkedin.com/in/rushichaganti/), support him by following.
 
-Follow the instructions in these guides to ensure proper configuration of **AIHawk** with **Ollama** and **Gemini**.
+- Auto-start AIHawk
```
Comment (surapuramakhil) [inline README.md:685]: make this as title (nesting accordingly) and for relavant

**Comment by feder-cr** [comment]
@sarob very good work

**Comment by sarob** [comment]
Thanks!Sean ***@***.*** On Oct 26, 2024, at 15:02, Federico ***@***.***> wrote:﻿
@sarob very good work

**Pull request merged**